articles/getting-started-with-game-development-using-windows-forms/index.md

```diff
@@ -136,18 +136,32 @@ After that, we need to check what the random method produced for each digit, and
             if (a == pink)
                 cool1.BackColor = Color.Pink;
             if (a == purple)
-                cool1.BackColor = Color.Purple;              
- //Repeat this with the snippet with the rest of the digits b c d.   
+                cool1.BackColor = Color.Purple;
+                
+                if (b == blue)
+                cool2.BackColor = Color.Blue;
+            if (b == red)
+                cool2.BackColor = Color.Red;
+            if (b == green)
+                cool2.BackColor = Color.Green;
+            if (b == yellow)
+                cool2.BackColor = Color.Yellow;
+            if (b == pink)
+                cool2.BackColor = Color.Pink;
+            if (b == purple)
+                cool2.BackColor = Color.Purple;
+
+ //Repeat this with the rest of the digits c & d.   
  
  ch1.Visible = true;
  //The previous line will turn the first Check button to Visible in order to start the first round.
         }        
 ```
-**NOTE** that you need to repeat the previous piece of code with `b` `c` and `d` before closing the method.
+**NOTE** that you need to repeat the previous piece of code with `c` and `d` before closing the method.
 
 
-Now we need to define the Check method that will check the colors chosed by the player and compare them with the colors produced by the system.
-The method will take the 4 colors chosed by the player the 4 colors produced by the system and the 4 hint digits as parameters.
+Now we need to define the Check method that will check the colors chosen by the player and compare them with the colors produced by the system.
+The method will take the 4 colors chosen by the player the 4 colors produced by the system and the 4 hint digits as parameters.

```

Comment:
change the 4 in numbers to four in wordings.
The method will take the four colors chosen by the player the four colors